choreolib/src/main/java/choreo/auto/AutoChooser.java

```diff
@@ -23,32 +27,41 @@
  * during auto start causing a delay.
  *
  * <p>Once the {@link AutoChooser} is made you can add {@link AutoRoutine}s to it using the {@link
- * #addAutoRoutine(String, AutoRoutineGenerator)} method. Unlike {@code SendableChooser} this
+ * #addAutoRoutine(String, AutoRoutineCreator)} method. Unlike {@code SendableChooser} this
  * chooser has to be updated every cycle by calling the {@link #update()} method in your {@link
  * IterativeRobotBase#robotPeriodic()}.
  *
  * <p>You can retrieve the {@link AutoRoutine} that is currently selected by calling the {@link
  * #getSelectedAutoRoutine()} method.
  */
 public class AutoChooser {
-  /** A function that generates an {@link AutoRoutine} from an {@link AutoFactory}. */
-  public static interface AutoRoutineGenerator extends Function<AutoFactory, AutoRoutine> {
+  /**
+   * A function that generates an {@link AutoRoutine} from an {@link AutoFactory}.
+   * Equivalent to a <code>{@literal Function<AutoFactory, AutoRoutine>}</code>.
+   */
+  public interface AutoRoutineCreator extends Function<AutoFactory, AutoRoutine> {
     /** A generator that returns an auto routine that does nothing */
-    static final AutoRoutineGenerator NONE = factory -> AutoFactory.VOID_ROUTINE;
+    static final AutoRoutineCreator NONE = factory -> AutoFactory.VOID_ROUTINE;
   }
 
+  /**
+   * A function that generates a {@link Command} from an {@link AutoFactory}.
+   * Equivalent to a <code>{@literal Function<AutoFactory, Command>}</code>.
+   */
+  public interface CommandCreator extends Function<AutoFactory, Command> {}
+
```

Comment:
idt this interface is needed

Reply:
Its to stay consistent w/ AutoRoutineCreator(fps prefers a separate functional interface so that custom docs can be added, I prefer just using Function)